src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -67,6 +67,14 @@
                 localStorage.setItem(profilerStorageKey + name, value);
             },
 
+            addEvent = function (element, eventName, callback) {
```

Comment:
I would call it ``addEventListener`` instead. It is adding a listener, not an event